mantidproject/mantid
# Reword engineering add peak button

**Description of work.**
Re-worded `Add Peak` button to read `Add Peak to List`

**To test:**
Open engineering gui and navigate to fitting tab, the button just above the `close` button should read `Add Peak to List`. Click `How to use` beneath Peak Tools, the tooltip should refer to the `Add Peak to List` button, not the `Add Peak button`

Fixes #25169 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
